custom_slack_provider/views.py

```diff
@@ -40,7 +55,7 @@ def get_data(self, token):
         if not resp.get('ok'):
             logger.exception(f'OAuth Exception: {resp.get("error")}')
             raise OAuth2Error()
-
+        
```

Comment:
this isn't PEP8 compliant -- with the empty tabbed line

Reply:
Remove extra spaces